tests/nat/tools/test_code_execution_sandbox.py

```
@@ -0,0 +1,447 @@
+
+    # Should return error for missing timeout field
+    result = response.json()
+    assert response.status_code == 400 and result["process_status"] == "error"
+
+
+@pytest.mark.integration
+def test_invalid_json(sandbox_config):
+    """Test request with invalid JSON."""
+    invalid_json = '{"generated_code": "print("test")", "timeout": 10}'
+
+    response = requests.post(sandbox_config["execute_url"],
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)